src/09_dictionaries.py

```diff
@@ -35,10 +35,20 @@
 
 # Add a new waypoint to the list
 # YOUR CODE HERE
+waypoints.append({"lat": 45, "lon": -121, "name": "a fourth place"})
+
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # YOUR CODE HERE
 
+waypoints[0]["lon"] = -130
+waypoints[0]["name"] = "not a real place"
```

Comment:
A good tip in programming in general, is to avoid the usage of magic numbers. In this case, when making the change... it may be better to change it based on the condition met, vs just a hardcoded magic number as the array index.
```
for waypoint in waypoints:
    if (waypoint["name"] == "a place"):
        waypoint.update({"lon": -130, "name": "not a real place"})
        break
```